python/lsst/dax/apdb/cassandra/apdbCassandra.py

```diff
@@ -811,7 +724,10 @@ def getDiaObjects(self, region: sphgeom.Region) -> pandas.DataFrame:
                 objects = cast(
                     pandas.DataFrame,
```

Comment:
pandas? (I know, pre-existing).

Reply:
The whole APDB API is around pandas. I have little love for it, and it is rather inefficient, but this what AP pipeline wants. 🤷